CHANGELOG.md

```diff
@@ -10,13 +10,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Added authority claimer service to support reader mode
+- Added `SFConfig`, `BHConfig` and `subscription_depth` configuration to `eth-input-reader`
```

Comment:
The internal names are not helpful for those who use the nome. We should add the full env var names here.

Reply:
Here is my proposal to the changelog:
```
### Changed
- Moved env vars A, B, C from state-fold config to eth-input-reader-config

### Removed
- Removed env vars A, B, C from eth-input-reader config
```